odoo/orm/decorators.py

```diff
@@ -45,17 +45,6 @@ def setter(method):
     return setter
 
 
-def propagate(method1: T | None, method2: T) -> T:
```

Comment:
Can you slit the commit and explain that it is not used since your PR on api.returns ?